.gitignore

```diff
@@ -1,2 +1,3 @@
+**.swp
```

Comment:
?

Reply:
Ignore SWP files that some text editors, such as Vim, leave behind.